pysph/sph/densitycorrection.py

```diff
@@ -0,0 +1,35 @@
+from pysph.sph.equation import Equation
```

Comment:
Maybe put this in the `wc` sub-package and the module could be called `density_correction.py`.